exporters/holstebro_decorate_leaders.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+"""
+Holstebro Kommune specific queries into MO.
+"""
+import json
+import logging
+import os
+import pathlib
+import queue
+import threading
+import time
+import string
+
+from datetime import datetime, timedelta
+
+import click
+import requests
+from anytree import Node, PreOrderIter
+
+import common_queries as cq
+import holstebro_helpers as hh
+import holstebro_logger
+from os2mo_helpers.mora_helpers import MoraHelper
+
+cfg_file = pathlib.Path.cwd() / 'settings' / 'holstebro.settings.json'
+if not cfg_file.is_file():
+    raise Exception('No setting file')
+SETTINGS = json.loads(cfg_file.read_text(encoding='utf-8'))
+
+logger = logging.getLogger('LederHierarki')
+
+
+@ click.command()
+@ click.option('--prod', 'hostname', required=True, flag_value='https://os2mo.holstebro.dk', help='MO Prod server')
+@ click.option('--test', 'hostname', required=True, flag_value='https://os2mo-test.holstebro.dk', help='MO Test server')
+@ click.option('--dev', 'hostname', required=True, flag_value='https://os2mo-dev.holstebro.dk', help='MO Dev server')
```

Comment:
Jeg tror der er røget et space ind imellem `@` og `click`:
```suggestion
@click.command()
@click.option('--prod', 'hostname', required=True, flag_value='https://os2mo.holstebro.dk', help='MO Prod server')
@click.option('--test', 'hostname', required=True, flag_value='https://os2mo-test.holstebro.dk', help='MO Test server')
@click.option('--dev', 'hostname', required=True, flag_value='https://os2mo-dev.holstebro.dk', help='MO Dev server')
```